### PR TITLE
[White Label] Backoffice: improve design for logo used in shopfront

### LINKED ISSUE #10902: [White Label] For uploading custom logo use same design as for enterprise logo/banner and provide required image dimensions
## What is the need / problem?
Uploading the custom logo uses a different design than uploading the enterprise logo or banner image.
Also the required image dimensions (in pixels) should be indicated.

Custom logo upload:
image

Enterprise logo and banner upload:
image

## Which type of users does this problem affect (and how many, if known)?
- Hub Managers (MCFEs)
- Instance Managers

## Potential solutions that will solve the problem
Use same design as on images tab in enterprise settings.

## Connected wishlist and discovery discussions
Follow up of [#10575](https://github.com/openfoodfoundation/openfoodnetwork/pull/10575).

### PR DESCRIPTION
**Pull request by jibees**
#### What? Why?

- Closes #10902

Capture d’écran 2023-05-31 à 10 40 16

#### What should we test?
- As a shop manager, go to your enterprise settings and white label panel
- Check that the design fits the requirements

#### Release notes

Changelog Category: User facing changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

File: app/views/admin/enterprises/form/_white_label.html.haml [hunk 1]
```diff
@@ -10,11 +10,13 @@
   .row
     .three.columns.alpha
       = f.label :white_label_logo, t('.upload_logo')
-    .three.columns
+      %br
+      217 x 44 pixels
```
Comment (mkllnk) [inline app/views/admin/enterprises/form/_white_label.html.haml:14]: Surprised that this isn't translated but nobody complained about it before. :shrug:

**Review by mkllnk — APPROVED** [review]

**Comment by filipefurtad0** [comment]
Hey @jibees ,
I just noticed some conflicts on this PR.
(Semaphore actually staged the PR - I wonder which code does it opt for...)

I'll re-stage/re-test once these have been addressed.

In any case, I've observed that:

Before this PR
image

After this PR
image

**Comment by jibees** [comment]
@filipefurtad0 Should be good. Worth a new testing, sorry...

**Review by mkllnk — APPROVED** [review]

**Comment by drummer83** [comment]
Hi @jibees,
Thanks for this upgrade! @filipefurtad0 I moved ahead and tested this even though your had already started because White Label is one of our priorities currently. Hope that's ok...

### Before staging
image

### After staging
image

Comparing to enterprise logo upload:
image

### Results
- The requested image size is indicated. ✔️ 
- The design is identical to the one for enterprise logo/banner upload. ✔️ 
- Functionality is still given as before. ✔️ 

Great! This one is ready to go! I will merge! 🎉 🏳️

**Pull request merged**